### DISCUSSION TITLE
Stuck on 404 /plugins/rpc/rpc.php with 4.3.3-3.2-r0

### DISCUSSION THREAD

**Discussion by sod75** [crazy-max/docker-rtorrent-rutorrent#371] [Q&A] [answered]
I update to latest and since then Sonarr won't connect anymore and gives the 404 not found issue
I've been trying to figure it out but I'm stumped to get a clear answer if this is now supposed to work out of the box?
I see messages about RPC no longer supported, but also that it is working, etc...

e.g. https://github.com/crazy-max/docker-rtorrent-rutorrent/issues/345 this mentions updating to latest should fix the problem.

Could anyone explain whether RPC
- should work as is?
- still works but needs a config change? ( if so what?)
- no longer works, but just change to ???, (how?)
- need to find another download client...

Thanks for any input..

![sonarr_err](https://github.com/crazy-max/docker-rtorrent-rutorrent/assets/4673458/066971f8-e34c-477f-a866-93bd7a10596a)
![rutorrentversion](https://github.com/crazy-max/docker-rtorrent-rutorrent/assets/4673458/0c45e8b6-ce14-406f-9493-5b3d65d7d67f)

**Comment by stickz** [comment, marked as answer]
We dropped support for RPC in `ruTorrent v4.3`. It's now required to point Sonarr to HTTPRPC. `/plugins/httprpc/action.php`. This will result in a faster and more reliable experience.

**Reply by sod75** [reply]
Thank you !!
That was easy, I saw the dropping of rpc, but nowhere on the net could I find the proposed alternative clearly.
